packages/medusa-payment-paypal/src/services/paypal-provider.ts

```diff
@@ -280,6 +281,26 @@ class PayPalProviderService extends AbstractPaymentProcessor {
     }
   }
 
+  async updatePaymentData(data, update) {
```

Comment:
todo: Correct me if I am wrong, but this method does not match the interface. The first argument should be `sessionId` and the second `data`. The implementations of Stripe and PayPal seem to be different because of this mix up.

Reply:
nice catch, updated